recipes/Chameleon/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "Chameleon" %}
+{% set version = "3.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Chameleon-{{ version }}.tar.gz
+  sha256: 52e0922702798486419c4d8bcb7d0b9d3e0629ee305c459180c688eb03ba445f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4
+  run:
+    - python !=3.0,!=3.1,!=3.2,!=3.3,!=3.4
+
+test:
+  imports:
+    - chameleon
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://chameleon.readthedocs.io
+  summary: Fast HTML/XML Template Compiler.
+  license: BSD
```

Comment:
Please specify which BSD license it is.

Reply:
on pypi it notes This software is made available as-is under a BSD-like license [1] (see included copyright notice).

[1] This software is licensed under the "Repoze" license.

I